docs/contribute.rst

```diff
@@ -112,40 +112,93 @@ Server architecture
 -------------------
 **Demos**
 
-- *URLs:*
-   - http://www-demo1.allizom.org/
-   - http://www-demo2.allizom.org/
-   - http://www-demo3.allizom.org/
-   - http://www-demo4.allizom.org/
-   - http://www-demo5.allizom.org/
-- *Bedrock locales:* dev repo
-- *Bedrock Git branch:* ``demo/1``, ``demo/2``, etc.
+Bedrock as a platform can run in two modes: Mozorg Mode (for content that will
+appear on mozilla.org) and Pocket Mode (for content that will end up on getpocket.com).
 
-These demos can be deployed by pushing a branch to one of the `demo/*` branches
-of the `mozilla/bedrock` repo. The `Github action`_ will then deploy the
-demo to Heroku.
+To support this, we have two separate sets of URLs we use for demos. To get code up to
+one of those URLs, push it to the specified branch on ``github.com/mozilla/bedrock``:
 
-.. _Github action: https://github.com/mozilla/bedrock/blob/main/.github/workflows/demo_deploy.yml
+- *Mozorg:*
+   - Branch ``mozorg-demo-1`` -> https://www-demo1.allizom.org/
+   - Branch ``mozorg-demo-2`` -> https://www-demo2.allizom.org/
+   - Branch ``mozorg-demo-3`` -> https://www-demo3.allizom.org/
+   - Branch ``mozorg-demo-4`` -> https://www-demo4.allizom.org/
+   - Branch ``mozorg-demo-5`` -> https://www-demo5.allizom.org/
+
+- *Pocket:*
+   - Branch ``pocket-demo-1`` -> https://www-demo1.tekcopteg.com/
+   - Branch ``pocket-demo-2`` -> https://www-demo2.tekcopteg.com/
+   - Branch ``pocket-demo-3`` -> https://www-demo3.tekcopteg.com/
+   - Branch ``pocket-demo-4`` -> https://www-demo4.tekcopteg.com/
+   - Branch ``pocket-demo-5`` -> https://www-demo5.tekcopteg.com/
+
+For example, for Mozorg:
 
 .. code-block:: bash
 
-  $ git push -f mozilla my-demo-branch:demo/1
+  $ git push -f mozilla HEAD:mozorg-demo-2
```

Comment:
Nit: maybe this is just personal preference, but `git push -f mozilla my-demo-branch:mozorg-demo-1` feels more explicit to me in terms of an example. Possibly less prone to an accidental wrong-branch push as well?

Reply:
Sure thing - will amend